Missing python in mta builder image

Hi,

it looks like the mta builder requires python (for db modules?).

I'm not sure if the best solution is to just install python in the image, as this will increase the image size by a lot, but I'm also not aware of a better solution.

Log output

```
$ docker run --rm -v "${PWD}":/project -it ppiper/mta-archive-builder:latest mtaBuild --mtar dummy.mtar --build-target CF build
SAP Multitarget Application Archive Builder 1.1.8
Module "myproject-db": /project/db/package.json already exists
Module "app": invoking npm install
Module "myproject-srv": invoking maven
Module "myproject-db": invoking npm
Module "myproject-db": command output
>
>  > @sap/hana-client@2.3.123 install /project/db/node_modules/@sap/hdi-deploy/node_modules/@sap/hana-client
>  > node build.js
>
>  { Error: Command failed: node-gyp configure
>  gyp ERR! configure error
>  gyp ERR! stack Error: Can't find Python executable "python", you can set the PYTHON env variable.
>  gyp ERR! stack     at PythonFinder.failNoPython (/opt/nodejs/node-v10.13.0-linux-x64/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:484:19)
>  gyp ERR! stack     at PythonFinder.<anonymous> (/opt/nodejs/node-v10.13.0-linux-x64/lib/node_modules/npm/node_modules/node-gyp/lib/configure.js:406:16)
>  gyp ERR! stack     at F (/opt/nodejs/node-v10.13.0-linux-x64/lib/node_modules/npm/node_modules/which/which.js:68:16)
>  gyp ERR! stack     at E (/opt/nodejs/node-v10.13.0-linux-x64/lib/node_modules/npm/node_modules/which/which.js:80:29)
>  gyp ERR! stack     at /opt/nodejs/node-v10.13.0-linux-x64/lib/node_modules/npm/node_modules/which/which.js:89:16
>  gyp ERR! stack     at /opt/nodejs/node-v10.13.0-linux-x64/lib/node_modules/npm/node_modules/isexe/index.js:42:5
>  gyp ERR! stack     at /opt/nodejs/node-v10.13.0-linux-x64/lib/node_modules/npm/node_modules/isexe/mode.js:8:5
>  gyp ERR! stack     at FSReqWrap.oncomplete (fs.js:154:21)
>  gyp ERR! System Linux 4.9.125-linuxkit
>  gyp ERR! command "/opt/nodejs/node-v10.13.0-linux-x64/bin/node" "/opt/nodejs/node-v10.13.0-linux-x64/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "configure"
>  gyp ERR! cwd /project/db/node_modules/@sap/hdi-deploy/node_modules/@sap/hana-client
>  gyp ERR! node -v v10.13.0
>  gyp ERR! node-gyp -v v3.8.0
>  gyp ERR! not ok
>
>      at ChildProcess.exithandler (child_process.js:289:12)
>      at ChildProcess.emit (events.js:182:13)
>      at maybeClose (internal/child_process.js:962:16)
>      at Process.ChildProcess._handle.onexit (internal/child_process.js:251:5)
>    killed: false,
>    code: 1,
>    signal: null,
>    cmd: 'node-gyp configure' } ''
>  npm WARN deploy@ No description
>  npm WARN deploy@ No repository field.
>  npm WARN deploy@ No license field.
>
>  npm ERR! code ELIFECYCLE
>  npm ERR! errno 255
>  npm ERR! @sap/hana-client@2.3.123 install: `node build.js`
>  npm ERR! Exit status 255
>  npm ERR!
>  npm ERR! Failed at the @sap/hana-client@2.3.123 install script.
>  npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
>
>  npm ERR! A complete log of this run can be found in:
>  npm ERR!     /home/mta/.npm/_logs/2019-04-01T14_35_41_039Z-debug.log
ERROR: An unexpected error has occurred; please tell us about it in our Community page: " https://answers.sap.com/tags/73554900100700001351"
```

--- a/mta-archive-builder/Dockerfile
+++ b/mta-archive-builder/Dockerfile
@@ -64,6 +64,10 @@ RUN apt-get update && apt-get install -y \
     INSTALLED_MAVEN_VERSION=`mvn -version |head -n1 |cut -d ' ' -f  3` && \
     echo "[INFO] maven version: \"${INSTALLED_MAVEN_VERSION}\"." && \
     #
+    # Install essential build tools and python, required for building db modules
+    #
+    apt-get install -y build-essential python-minimal && \
+    #
     # Cleanup curl (was only needed for downloading artifacts)
     #
     apt-get remove --purge --autoremove -y \

